### DISCUSSION TITLE
syntax for multiple where clauses

### DISCUSSION THREAD

**Discussion by trondhindenes** [piccolo-orm/piccolo#961] [General]
I just realized that this doesn't seem to be documented. Probably a good idea to add it.

**Comment by dantownsend** [comment]
There's a section here, but it's buried pretty deep in the docs:

https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_clauses/where.html?highlight=where#complex-queries-and-or

You're right - the `Complex queries - and / or` section should be broken down into subsections. One could be `Multiple Where Clauses`, so it appears in search.

**Reply by dantownsend** [reply]
I broke up the `where` clause docs - adding a new sub section. Hopefully it's a bit better now.

https://piccolo-orm.readthedocs.io/en/latest/piccolo/query_clauses/where.html?highlight=where#multiple-where-clauses

**Reply by trondhindenes** [reply]
thanks! I learned a few new things from that update, nice :-)
